utilities/openemr-env-installer/openemr-env-installer

```diff
@@ -275,12 +300,63 @@ else
 fi
 }
 
+# Install command check
+install_cmd_chk(){
+	which install &>/dev/null
+	if [ $? -ne 0 ]
+	then
+		echo -e "\033[33mNOTE: Please make sure you have installed coreutils package at first.\033[0m"
+		exit
+	fi
+}
+
+# Run minikube cmd to startup cluster tip
+run_cluster_tip() {
+	echo -e "\033[32mFrom a terminal with administrator access (but not logged in as root), run:\033[0m"
+	echo -e "\033[36m     minikube start \033[0m"
+	echo -e "\033[32mIf minikube fails to start, see the drivers page(https://minikube.sigs.k8s.io/docs/drivers/)\033[0m"
+	echo -e "\033[32mfor help setting up a compatible container or virtual-machine manager.\033[0m"
+}
+
+# Install minikube
+install_minikube(){
+# Judge the os type e.g. Linux or MacOS
+if [ "$os_type" == "Linux" ]
+then
+	minikube_os=linux
+elif [ "$os_type" == "Darwin" ]
+then
+	minikube_os=darwin
+fi
+	# Install minikube
+	which minikube &>/dev/null
+	if [ $? -ne 0 ]
+	then
+		echo -e "\033[30m===>\033[0m \033[32mInstalling minikube... \033[0m"
+		echo ''
+		#curl -LO https://storage.googleapis.com/minikube/releases/latest/minikube-${minikube_os}-amd64
```

Comment:
if this is commented out, then how is it getting the minikube package?

Reply:
@bradymiller sorry, due to network issue for some test vm, so I comment out for test and forgot to restore